src/Playwright/Helpers/Paths.cs

```diff
@@ -34,18 +35,55 @@ internal static class Paths
         internal static string GetExecutablePath()
         {
             var assembly = typeof(Playwright).Assembly;
-            string driversPath = new FileInfo(assembly.Location).Directory.FullName;
+            string driversPath, executableFile;
 
-            string executableFile = GetPath(driversPath);
+            // in .NET 6, the notion of single-assembly apps means, we won't have a location
```

Comment:
It is unlikely that we support single-assembly apps, because we require driver and browser, so I would remove this and close single-assembly as Won't fix.

Reply:
The latest change should make them supported - I added a separate build target for `Publish` which copies over the `.playwright` folder. That said, I did add an `#ifdef`, but it also makes our discovery easier.